opencv_engine/engine.py

```
@@ -286,3 +288,14 @@ def flip_horizontally(self):
         """ flip an image horizontally (about y-axis) """
         image = numpy.asarray(self.image)
         self.image = cv.fromarray(cv2.flip(image, 1))
+
+    def _get_exif_segment(self):
+        """ Override because the superclass doesn't check for no exif.
+        """
+        segment = None
+        try:
+            if getattr(self, 'exif', None) is not None:
+                segment = ExifSegment(None, None, self.exif, 'ro')
+        except Exception:
+            logger.warning('Ignored error handling exif for reorientation', exc_info=True)
+        return segment
```

Is this better style than

`""" docstring"""
         try:
             if getattr(self, 'exif', None) is not None:
                 return ExifSegment(None, None, self.exif, 'ro')
         except Exception:
             logger.warning('Ignored error handling exif for reorientation', exc_info=True)
             return None